src/Routers/AggregateRouter.js

```diff
@@ -34,6 +34,8 @@ const ALLOWED_KEYS = [
   'unwind',
 ];
 
+const ALLOWED_KEYS = BASE_KEYS.concat(PIPELINE_KEYS);
```

Comment:
This will mutate BASE_KEYS IIRC :)

Reply:
Concat docs say it doesn’t mutate

Reply:
Ok my bad, getting a bit rusty :p spread is quite nice :)